doc/whats_new.rst

```diff
@@ -40,6 +40,8 @@ Changelog
 
 - Add :func:`~hnn_core.Network.pick_connection` to query the indices of specific connections in :attr:`~hnn_core.Network.connectivity`, by `Nick Tolley`_ in `#367 <https://github.com/jonescompneurolab/hnn-core/pull/367>`_
 
+- Unify the codepath for instantiating drive and local network connectivity, by `Ryan Thorpe`_, `Mainak Jas`_, and `Nick Tolley`_ in `#369 <https://github.com/jonescompneurolab/hnn-core/pull/369>`_
```

Comment:
You should think of "what affects the user" when updating this. User does not care about code path. They care about:

- change in behavior (e.g., with 0 weights in param files, you now have empty connections?). That's a bug fix.
- change in `net.connectivity`. It has more elements now and probably not in the same order. If someone hard coded `net.connectivity[32]` it would now break. So that's an API change.

Reply:
What do you think of https://github.com/jonescompneurolab/hnn-core/pull/369/commits/36638267e792da935d82b69ce3ce01733c4f492f @jasmainak?

Reply:
Looks good!